07-Customizing-AzSK-for-your-Org/Readme.md

```diff
@@ -1555,6 +1555,50 @@ v) Override ControlSettings.json in ServerConfigMetaData.json as shown below:
 
 vi) Rerun the policy update or setup command (the same command you ran for the first-time setup).
 
+### How can we treat each public IP address as an individual resource?
+   Org policy admins can enable public IP address as an individual resource by making following org policy settings:
+
+i) Copy the ControlSettings.json from the AzSK installation to your org-policy folder.
+
+ii) Update the EnablePublicIpResource setting to true.
+```JSON
+"PublicIpAddress": {
+    "EnablePublicIpResource": true
+   },
+```
+
+iii) Save the file.
+
+iv)  Edit the ServerConfigMetadata.json file in your local org-policy folder and create an entry for "ControlSettings.json" as below:
```

Comment:
what if admins have overridden the control settings.

Reply:
Added the instructions for already overridden ControlSettings.json in step no.1a and step no.5.